backend/config.example.js

```diff
@@ -12,6 +12,9 @@ module.exports = {
     cert: '/path/to/root_ca_cert.crt'
   },
   login: {
+    adminRole: 'some-role',
+    internRole: 'some-other-role',
```

Comment:
What is `internRole` exactly? Remember that "intern" in English is "Praktikant" in German...